packages/mds-web-sockets/launch_server.ts

```diff
@@ -1,3 +1,6 @@
 import { WebSocketServer } from './server'
 
 WebSocketServer()
+  .then(() => console.log(`WebSockerServer running`))
```

Comment:
Should rename this file to just `server.ts` and use it as the container-image entry point for webpack.